### PR TITLE
fix: #24330

### PR DESCRIPTION
**Pull request by TheLeoP**

### CODE REVIEW

**Comment by TheLeoP** [comment]
@famiu

File: src/nvim/ex_getln.c [hunk 1]
```diff
@@ -2409,6 +2409,32 @@ static void cmdpreview_prepare(CpInfo *cpinfo)
 static void cmdpreview_restore_state(CpInfo *cpinfo)
   FUNC_ATTR_NONNULL_ALL
 {
+  Set(ptr_t) saved_bufs = SET_INIT;
+  Set(ptr_t) saved_wins = SET_INIT;
```
Comment (famiu) [inline src/nvim/ex_getln.c:2413]: Why do you need the sets? The windows and buffers are already guaranteed to be unique.

Comment (TheLeoP) [reply src/nvim/ex_getln.c:2413]: I didn't want to iterate over all the saved buffer and windows each time

Comment (famiu) [reply src/nvim/ex_getln.c:2413]: Oh, nevermind, I misread the code. My bad.

**Pull request closed without merging**